From d5df26e019e8b6c1c76a00617d350692b9d69ba3 Mon Sep 17 00:00:00 2001
From: Valentin Raevsky <valentin@compulab.co.il>
Date: Wed, 29 Apr 2020 21:10:22 +0300
Subject: [PATCH 42/42] cl-som-imx8: defconfig: Update config

1) Enabled
CONFIG_BLK_DEV_RAM
CONFIG_HW_RANDOM
CONFIG_CRYPTO_RSA
CONFIG_CRYPTO_AUTHENC
CONFIG_CRYPTO_DES

2) Changed Y->M
CONFIG_CRYPTO_DEV_FSL_CAAM
CONFIG_CRYPTO_DEV_FSL_CAAM_SECVIO

3) Disabled
CONFIG_MMC_SDHCI_ACPI

Signed-off-by: Valentin Raevsky <valentin@compulab.co.il>
---
 arch/arm64/configs/cl-som-imx8_defconfig | 10 +++++++---
 1 file changed, 7 insertions(+), 3 deletions(-)

diff --git a/arch/arm64/configs/cl-som-imx8_defconfig b/arch/arm64/configs/cl-som-imx8_defconfig
index 9b4033f..7a38120 100644
--- a/arch/arm64/configs/cl-som-imx8_defconfig
+++ b/arch/arm64/configs/cl-som-imx8_defconfig
@@ -150,6 +150,7 @@ CONFIG_MTD_UBI=y
 CONFIG_OF_OVERLAY=y
 CONFIG_BLK_DEV_LOOP=y
 CONFIG_BLK_DEV_NBD=m
+CONFIG_BLK_DEV_RAM=y
 CONFIG_XEN_BLKDEV_BACKEND=m
 CONFIG_VIRTIO_BLK=y
 CONFIG_SENSORS_FXOS8700=y
@@ -224,6 +225,7 @@ CONFIG_SERIAL_XILINX_PS_UART_CONSOLE=y
 CONFIG_SERIAL_FSL_LPUART=y
 CONFIG_SERIAL_FSL_LPUART_CONSOLE=y
 CONFIG_VIRTIO_CONSOLE=y
+CONFIG_HW_RANDOM=y
 CONFIG_I2C_CHARDEV=y
 CONFIG_I2C_MUX_PCA954x=y
 CONFIG_I2C_DESIGNWARE_PLATFORM=y
@@ -389,7 +391,6 @@ CONFIG_MMC=y
 CONFIG_MMC_BLOCK_MINORS=32
 CONFIG_MMC_ARMMMCI=y
 CONFIG_MMC_SDHCI=y
-CONFIG_MMC_SDHCI_ACPI=y
 CONFIG_MMC_SDHCI_PLTFM=y
 CONFIG_MMC_SDHCI_ESDHC_IMX=y
 CONFIG_MMC_SPI=y
@@ -491,13 +492,16 @@ CONFIG_MEMTEST=y
 CONFIG_SECURITY=y
 CONFIG_SECURITY_NETWORK=y
 CONFIG_SECURITY_SELINUX=y
+CONFIG_CRYPTO_RSA=y
+CONFIG_CRYPTO_AUTHENC=y
 CONFIG_CRYPTO_ECHAINIV=y
 CONFIG_CRYPTO_SHA512=m
+CONFIG_CRYPTO_DES=y
 CONFIG_CRYPTO_ANSI_CPRNG=y
-CONFIG_CRYPTO_DEV_FSL_CAAM=y
+CONFIG_CRYPTO_DEV_FSL_CAAM=m
 CONFIG_CRYPTO_DEV_FSL_CAAM_SM=y
 CONFIG_CRYPTO_DEV_FSL_CAAM_SM_TEST=y
-CONFIG_CRYPTO_DEV_FSL_CAAM_SECVIO=y
+CONFIG_CRYPTO_DEV_FSL_CAAM_SECVIO=m
 CONFIG_ARM64_CRYPTO=y
 CONFIG_CRYPTO_SHA1_ARM64_CE=y
 CONFIG_CRYPTO_SHA2_ARM64_CE=y
-- 
1.9.1

